spec/controllers/posts_controller_spec.rb

```diff
@@ -106,12 +107,6 @@
         expect(assigns(:search_results)).to match_array([post1, post2])
       end
 
-      it "does not mix up subject with content" do
-        create(:post, subject: 'unrelated', content: 'contains stars')
```

Comment:
Shouldn't we still have this test, just adding a reply in?

Reply:
I wouldn't expect that to be a thing it could be confused about, with a reply?

Reply:
Probably not! Hmmmmmmmm. Okay I'm fine with this out.